package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "reviews",
+  "description": "",
+  "version": "1.0.0",
+  "main": "ETL.js",
+  "scripts": {
+    "etl": "node ETL.js"
+  },
+  "dependencies": {
+    "csv-parser": "^3.0.0",
+    "dotenv": "^16.3.1",
+    "express": "^4.18.2",
+    "mongodb": "^6.3.0",
+    "mongoose": "^8.0.2",
+    "nodemon": "^3.0.2",
+    "postgres": "^3.4.3"
```

Comment:
make sure to remove unneeded dependencies at some point. (I think just csv-parser, mongodb, and mongoose)

Reply:
actually I thought about this a bit more while working on it, if youd like to leave the extra dependencies that should be fine, i am going to leave a couple extra on mine so that i can see them afterwards when i look back at the project